src/components/chat/AddDirectMessageChannel.tsx

```diff
@@ -36,7 +33,9 @@ const useStyles = makeStyles((theme) => ({
 
 const AddDirectMessageChannel: React.FC = () => {
   const classes = useStyles();
+
```

Comment:
Bitte keine leeren Zeilen, wo vorher keine waren. Macht den Code nicht übersichtlicher, ich mach das nie so und hab auch nicht vor, das in Zukunft so zu machen. :-)